airflow/models/abstractoperator.py

```diff
@@ -115,7 +115,7 @@ def get_template_env(self) -> "jinja2.Environment":
 
         dag = self.get_dag()
         if dag:
-            return dag.get_template_env()
+            return dag.get_template_env(force_string=True)
```

Comment:
When is this False?

Reply:
Default as False

Reply:
Oh, right 😓

Reply:
I don’t think this is correct though. This function is also used when the task is being executed (in `render_template_fields`), and in _that_ situation we want to use `NativeEnvironment` if `render_template_as_native_obj` is True. so this `get_template_env` also need to have a `force_string` argument, and only `get_email_subject_content` should set it to True.